Error with in-source compilation: `cmake .  && make && cmake . && make`

I an traing  to build cgal on ubuntu 16.4 and i am facing this type of error for all g++ > 4.8 

[  0%] Built target CGAL
[  0%] Building CXX object Installation/src/CGAL_Qt5/CMakeFiles/CGAL_Qt5.dir/all_files.cpp.o
In file included from /data_local/CGAL/cgal-master/build/Installation/src/CGAL_Qt5/all_files.cpp:2:0:
/data_local/CGAL/cgal-master/Installation/src/CGAL_Qt5/qrc_Triangulation_2.cpp:9:45: error: redefinition of ‘const unsigned char qt_resource_data []’
 static const unsigned char qt_resource_data[] = {


Any idea

Thank's
                                             ^

--- a/Installation/src/CMakeLists.txt
+++ b/Installation/src/CMakeLists.txt
@@ -7,7 +7,11 @@ function (collect_cgal_library LIBRARY_NAME ADDITIONAL_FILES)
   # THEN collect *.cpp
   foreach (package ${CGAL_CONFIGURED_PACKAGES} )
     file(GLOB CGAL_LIBRARY_SOURCE_FILES_TMP ${package}/src/${LIBRARY_NAME}/*.cpp)
-    list(APPEND CGAL_LIBRARY_SOURCE_FILES ${CGAL_LIBRARY_SOURCE_FILES_TMP})
+    foreach(file ${CGAL_LIBRARY_SOURCE_FILES_TMP})
+      if(NOT ${file} MATCHES "/qrc_.*")
+        list(APPEND CGAL_LIBRARY_SOURCE_FILES ${file})
+      endif()
+    endforeach()
   endforeach()
 
   foreach(source ${CGAL_LIBRARY_SOURCE_FILES})

